db.ts

```diff
@@ -3,45 +3,41 @@ import IProduct from '@packages/entities/IProduct';
-    price: 10,
-    discount: 50,
-    description:
-      'It is a long established fact that a reader will be distracted.',
+    title: 'Caneca PodCodar - Céu Estrelado',
+    img: '/images/products/caneca02-ceu02.png',
+    price: 35,
+    discount: 14,
+    description: 'Caneca PodCodar com estampa do Céu Estrelado',
     fullDescription:
       'It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using Content here, content here, making it look like readable English. ',
   },
   {
     id: 3,
-    title: 'Camiseta 3',
-    img: 'https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR6XGdJw5NsYRRklb8YpamARtn8YTG5aOXVhg&usqp=CAU',
-    price: 10,
-    discount: 0,
-    description:
-      'It is a long established fact that a reader will be distracted.',
+    title: 'Caneca PodCodar - Lhama Rosa',
+    img: '/images/products/caneca03-lhama-rosa03.png',
+    price: 35,
+    discount: 14,
+    description: 'Caneca PodCodar com estampa da Lhama e fundo Rosa',
```

Comment:
acho que se remover o `e fundo Rosa` o css não quebra. Quick and dirty fix 😆

Reply:
Removido